src/scripts/main.js

```diff
@@ -1,3 +1,260 @@
 'use strict';
 
-// write your code here
+let board;
+const rowsCount = 4;
+const columnsCount = 4;
+const score = document.querySelector('.game-score');
```

Comment:
the score is always 0:
![image](https://user-images.githubusercontent.com/86923089/209211246-815dd597-47c1-4f87-86e6-0014513820bf.png)